src/bench_ecmult.c

```diff
@@ -123,6 +124,31 @@ static void bench_ecmult_const_teardown(void* arg, int iters) {
     bench_ecmult_teardown_helper(data, &data->offset1, &data->offset2, NULL, iters);
 }
 
+static void bench_ecmult_const_xonly(void* arg, int iters) {
+    bench_data* data = (bench_data*)arg;
+    int i;
+
+    for (i = 0; i < iters; ++i) {
+        const secp256k1_ge* pubkey = &data->pubkeys[(data->offset1+i) % POINTS];
+        const secp256k1_scalar* scalar = &data->scalars[(data->offset2+i) % POINTS];
+        secp256k1_ecmult_const_xonly(&data->output_xonly[i], &pubkey->x, NULL, scalar, 1);
+    }
+}
+
+static void bench_ecmult_const_xonly_teardown(void* arg, int iters) {
+    bench_data* data = (bench_data*)arg;
+    int i;
+
+    /* verify by comparing with x coordinate of regular ecmult_const result */
+    for (i = 0; i < iters; ++i) {
+        const secp256k1_ge* pubkey = &data->pubkeys[(data->offset1+i) % POINTS];
+        const secp256k1_scalar* scalar = &data->scalars[(data->offset2+i) % POINTS];
+        secp256k1_gej expected_gej;
+        secp256k1_ecmult_const(&expected_gej, pubkey, scalar);
+        CHECK(secp256k1_gej_eq_x_var(&data->output_xonly[i], &expected_gej));
+    }
```

Comment:
Is there a reason you don't use the faster `ecmult` function, like so?
```suggestion
    for (i = 0; i < iters; ++i) {
        const secp256k1_ge* pubkey = &data->pubkeys[(data->offset1+i) % POINTS];
        const secp256k1_scalar* scalar = &data->scalars[(data->offset2+i) % POINTS];
        secp256k1_gej pubkey_gej;
        secp256k1_gej expected_gej;
        secp256k1_gej_set_ge(&pubkey_gej, pubkey);
        secp256k1_ecmult(&expected_gej, &pubkey_gej, scalar, NULL);
        CHECK(secp256k1_gej_eq_x_var(&data->output_xonly[i], &expected_gej));
    }
```